gen/BellArch_gen.ml

```diff
@@ -271,6 +271,7 @@ include
       let is_symbolic _ = false
 
       let pp_reg = pp_reg
+      let pp_i _ = "instr:\"NOP\""
```

Comment:
```suggestion
      let pp_i _ = assert false
```
Does this function need to have an implementation for non-AArch64? My hunch is that it should never be called unless this is an error.